evap/contributor/views.py

```diff
@@ -135,8 +134,12 @@ def course_edit(request, course_id):
         if post_operation not in ('save', 'approve'):
             raise SuspiciousOperation("Invalid POST operation")
 
-        course_form.save(user=user)
-        formset.save()
+        if course_form.has_changed():
+            course_form.save(user=request.user)
+        elif formset.has_changed():
```

Comment:
This `elif` will break if a user changes both, the `formset` and the `course_form`. You probably want something like:

```
if course_form.has_changed() or formset.has_changed():
    # Save form, even if only formset has changed, to update last_modified_user
    course_form.save(user=request.user)

if formset.has_changed():
    formset.save()
```

Also, since we are obivously missing a tests for this case, would you mind adding one? Otherwise, I would add one, if you don't want to.

Reply:
Hey, thanks for the review! you are totally right, I flipped around the order of the conditions here 😬 I changed it to 
```
if formset.has_changed():
    formset.save()
    # Save form, even if only formset has changed, to update last_modified_user
    form.save(user=request.user)
elif form.has_changed():
    form.save(user=request.user)
```
which should be equivalent to your proposal, right?

I also added tests for both cases (formset changed, nothing changed)